File: pdlearning/src/web-apps/src/libraries/domain-api/src/access-right/models/access-right.ts
export interface IAccessRight {
  id: string;
  objectId: string;
  userId: string;
  createdDate: string;
}

export class AccessRight implements IAccessRight {
  public id: string;
  public objectId: string;
  public userId: string;
  public createdDate: string;

  constructor(data?: IAccessRight) {
    if (data == null) {
      return;
    }
    this.id = data.id;
    this.objectId = data.objectId;
    this.userId = data.userId;
    this.createdDate = data.createdDate;
  }
}

File: pdlearning/src/microservices/form-microservice/Api/Domain/Entities/AccessRight.cs
using System;
using Thunder.Platform.Core.Domain.Auditing;

namespace Microservice.Form.Domain.Entities
{
    public class AccessRight : AuditedEntity
    {
        public Guid UserId { get; set; }

        /// <summary>
        /// The digital content origial object Id.
        /// </summary>
        public Guid ObjectId { get; set; }

        public Guid CreatedBy { get; set; }
    }
}
